webapp/src/js/services/target-aggregates.js

```diff
@@ -0,0 +1,257 @@
+const moment = require('moment');
+const _ = require('lodash');
+
+angular.module('inboxServices').factory('TargetAggregates',
+  function(
+    $q,
+    Auth,
+    CalendarInterval,
+    ContactTypes,
+    DB,
+    GetDataRecords,
+    Search,
+    Settings,
+    UHCSettings,
+    UserSettings
+  ) {
+
+    'use strict';
+    'ngInject';
+
+    const getCurrentIntervalTag = (settings) => {
```

Comment:
This function could use some documentation like an example of what's returned and why we use UHC settings.

Reply:
I added some comments, please tell if they're clear enough.

Reply:
Works for me! Thanks.